src/runtime/ui-slot-composer.ts

```diff
@@ -233,7 +233,7 @@ export class UiSlotComposer {
     log('sendEvent:', particleId, eventlet);
     const findConsumer = id => this.consumers.find(
         consumer => consumer.consumeConn.particle.id.toString() === id);
-    const consumer = findConsumer(particleId);
+    const consumer = findConsumer(particleId.toString());
```

Comment:
can you, please, types to this method? i was actually thinking that particleId here is expected to be string, and this is what i'm passing from uiEvent (in run-arc.js, happy to adapt, let's just coordinate).
otoh i think that from verbs/event.js you are passing the entire particle object...

Reply:
Fair point. I'll take a stab at it.

Reply:
Fwiw, `verbs/event.js` is invoking `arc.pec.sendEvent` which is different API altogether (however, this is itself confusing).